scripts/install/get_tendermint.sh

```diff
@@ -1,51 +1,47 @@
-#!/bin/bash
+#!/usr/bin/env bash
 
-#set -x
+set -euEo pipefail
```

Comment:
with ubuntu `GNU bash, version 5.0.17(1)-release (x86_64-pc-linux-gnu)`, this only seems to work with:
```suggestion
set -Eo pipefail
```